### DISCUSSION TITLE
How to export audio based on my region duration

### DISCUSSION THREAD

**Discussion by nahansans** [katspaugh/wavesurfer.js#2974] [Q&A]
I have audio with 4 minutes duration, I only want to export 30 seconds of audio by region.

**Comment by katspaugh** [comment]
I already replied to you [here](https://github.com/katspaugh/wavesurfer.js/issues/2591#issuecomment-1620606848):

You can extract a portion using .slice() on the array.

Something like

```js
const audioData = wavesurfer.getDecodedData()
const channel = audioData.getChannelData(0)
const portion = channel.slice(region.start * audioData.sampleRate, region.end * audioData.sampleRate)
```
